app/pages/project/project-page.cjsx

```diff
@@ -164,6 +171,18 @@ ProjectPage = React.createClass
           </Link>
         }
 
+        {if @props.user && this.userHasLabAccess()
```

Comment:
Just had an idea, this could be one extra tab with Icons similar to how the social links work.